doc/usage.rst

```diff
@@ -89,34 +89,79 @@ can be updated using the method update_flags.
-                     loop_start,
-                         Instr('FOR_ITER', loop_done),
-                         Instr('STORE_NAME', 'x'),
-                         Instr('LOAD_NAME', 'print'),
-                         Instr('LOAD_NAME', 'x'),
-                         Instr('CALL_FUNCTION', 1),
-                         Instr('POP_TOP'),
-                         Instr('JUMP_ABSOLUTE', loop_start),
-                     loop_done,
-                         Instr('POP_BLOCK'),
-                     loop_exit,
-                         Instr('LOAD_CONST', None),
-                         Instr('RETURN_VALUE')])
-
-    # the conversion to Python code object resolve jump targets:
-    # replace abstract labels with concrete offsets
-    code = code.to_code()
-    exec(code)
+Bytecode of ``for x in (1, 2, 3): print(x)``:
+
+.. tabs::
+
+    .. group-tab:: Python < 3.8
+
+        .. code:: python
+
+            from bytecode import Label, Instr, Bytecode
+
+            loop_start = Label()
+            loop_done = Label()
+            loop_exit = Label()
+            code = Bytecode(
+                [
+                    Instr('SETUP_LOOP', loop_exit),
+                    Instr('LOAD_CONST', (1, 2, 3)),
+                    Instr('GET_ITER'),
+                    loop_start,
+                        Instr('FOR_ITER', loop_done),
+                        Instr('STORE_NAME', 'x'),
+                        Instr('LOAD_NAME', 'print'),
+                        Instr('LOAD_NAME', 'x'),
+                        Instr('CALL_FUNCTION', 1),
+                        Instr('POP_TOP'),
+                        Instr('JUMP_ABSOLUTE', loop_start),
+                    loop_done,
+                        Instr('POP_BLOCK'),
+                    loop_exit,
+                        Instr('LOAD_CONST', None),
+                        Instr('RETURN_VALUE')
+                ]
+            )
+
+            # the conversion to Python code object resolve jump targets:
+            # replace abstract labels with concrete offsets
+            code = code.to_code()
+            exec(code)
+
+    .. group-tab:: Python >= 3.8
+
+        .. code:: python
+
+            from bytecode import Label, Instr, Bytecode
+
+            loop_start = Label()
+            loop_done = Label()
+            loop_exit = Label()
+            code = Bytecode(
+                [
+                    # Python 3.8 removed SETUP_LOOP
+                    Instr("LOAD_CONST", (1, 2, 3)),
+                    Instr("GET_ITER"),
+                    loop_start,
+                        Instr("FOR_ITER", loop_exit),
+                        Instr("STORE_NAME", "x"),
+                        Instr("LOAD_NAME", "print"),
+                        Instr("LOAD_NAME", "x"),
+                        Instr("CALL_FUNCTION", 1),
+                        Instr("POP_TOP"),
+                        Instr("JUMP_ABSOLUTE", loop_start),
+                    # Python 3.8 removed the need to manually manage blocks
```

Comment:
Only for loops. For `with` and `try` you need to add some `POP_BLOCK`s.

Reply:
Thanks for pointing this out @serhiy-storchaka, I will rephrase the comment.

I need to add documentation for those other cases.